parley/src/font.rs

```diff
@@ -6,9 +6,18 @@ use fontique::Collection;
 #[cfg(feature = "std")]
 use fontique::SourceCache;
 
+/// A font database/cache (wrapper around a the [fontique] crate)
+///
+/// This type is designed to be a global resource with only one per-application (or per-thread)
 #[derive(Default)]
 pub struct FontContext {
     pub collection: Collection,
     #[cfg(feature = "std")]
     pub source_cache: SourceCache,
 }
+
+impl FontContext {
+    pub fn new() -> Self {
```

Comment:
Given that this does (iirc) search the system font libraries, we should have a doc comment here.

Reply:
```suggestion
    /// Create a new `FontContext`, discovering system fonts if available.
    pub fn new() -> Self {
```

Reply:
Obviously this API could use some work, but that's not the job of this PR.